gulpfile.js

```diff
@@ -2,9 +2,9 @@ var gulp = require('gulp');
 var gulpif = require('gulp-if');
 var del = require('del');
 var dom = require('gulp-dom');
-var uglifyes = require('uglify-es');
+var terser = require('terser');
 var composer = require('gulp-uglify/composer');
-var uglify = composer(uglifyes, console);
+var minify = composer(terser, console);
```

Comment:
What's the reason for naming it `minify`? I would rather call it `tersify`, as minification should be a different process.